machine-learning/_posts/2017-03-30-GMM.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: review
+title:  "Learning patterns of activity using real-time tracking"
+tags:   motion-detection segmentation essentials
+author: Yi Wang
+pdf:    http://profs.sci.univr.it/~cristanm/teaching/sar_files/lezione4/vsam-pami-tracking.pdf
+cite:
+  authors: "Stauffer, Chris and Grimson, W. Eric L."
+  title:   "Learning patterns of activity using real-time tracking"
+  venue:   "PAMI, 2000"
+---
+
+Gaussian mixture model (GMM) is one of the most popular motion detection models. As a parametric models, GMM generates multiple Gaussian models for each pixel in the video. When a new frame arrives, GMM compares each pixel in the frame with every Gaussian model in the mixture until a matching Gaussian model is found. The matched Gaussian will be updated, and the pixel will be considered to be a background. If no Gaussian model mateched, the least probable distribution will be replaced with a distribution with its mean equales to the current pixel value, and the pixel will be considered to be a foreground.
```

Comment:
Plaese add a GMM tag in the 2017-03-30-GMM.md file as well as in the how-to file.